packages/tools/devtools/devtools-browser-extension/tsconfig.json

```diff
@@ -1,14 +1,14 @@
 {
 	"extends": "@fluidframework/build-common/ts-common-config.json",
 	"compilerOptions": {
-		"rootDir": "src",
+		"target": "es2017",
+		"module": "esnext",
```

Comment:
Let's keep this file as it was before. We have a pattern across the repo for packages that have test resources which need to be built. Look at the `packages/dds/shared-object-base` as an example; we'll want a separate `tsconfig.json` file inside the folder with the test resources (for shared-object-base that's `src/test`, in this case it's `e2e-tests/`) that only includes those files, and a `build:test` npm script that points to that file instead of the main `tsconfig.json`.

Also, what makes the `target` change necessary?

Reply:
Eslint was warning that to use await in .tsx. Target must be later than 2017.

Reply:
This file was already targeting es2018, thus my question. I think the complaint it was about the `"module": "esnext"` bit which is indeed required for top-level await, but the `target` probably doesn't need to change for that (particularly, it shouldn't need to go back to 2017, I don't think).